Protactor/conf.js

```diff
@@ -22,10 +22,15 @@ exports.config = {
     baseUrl: 'https://www.saucedemo.com',
     jasmineNodeOpts: {
         timeout: 1000 * 60 * 5,
+        defaultTimeoutInterval: 180000,
     },
-    onComplete(passed) {
-        return browser.getSession().then(
-            session => saucelabs.updateJob(session.getId(), { passed }),
-        )
-    },
+
+    onComplete() {
+        var printSessionId = (jobName) => {
```

Comment:
Hmm, I had thought maybe it was used by something since this was in one of our other example framework source but it looks like that itself may have been an improper copy.